src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2052,12 +2052,17 @@ def config_source_control(cmd, resource_group_name, name, repo_url, repository_t
 def update_git_token(cmd, git_token=None):
     '''
     Update source control token cached in Azure app service. If no token is provided,
-    the command will clean up existing token.
+    the command will clean up existing token. Note that tokens are now redacted in the result.
```

Comment:
Question for reviewer: I do not see any help.py file with this text. Does that mean that this is the text used for the documentation [here](https://learn.microsoft.com/en-us/cli/azure/functionapp/deployment/source?view=azure-cli-latest#az-functionapp-deployment-source-update-token)?

Reply:
yes, correct